docs/docs/how-tos/jupyter-gallery.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Share Git Repository via JupyterLab-Gallery
+description: Share files or tutorials with your team
+---
+
+# How to set up a Git Repository via JupyterLab-Gallery
+
+[JupyterLab-Gallery ](https://github.com/nebari-dev/jupyterlab-gallery) is a JupyterLab plugin
+that allows users to share a Git Repository. JupyterLab users are presented with a tile in
+the JupyterLab launch screen. Users can choose to download the Gallery.
+
+By default, if no JupyterLab-Gallery items are added to the config, users will not see this section
+in the JupyterLab Launcher.
+
+This feature was added to Nebari in the 2024.6.1 release.
+
+## Configure JupyterLb-Gallery
+
+In Nebari, the JupyterLab-Gallery configuration is set in the Nebari configuration, `nebari-config.yaml`.
+
+The configuration is set under the `jupyterlab` section under the subsection `gallery_settings`.
+Users may set a title for the Gallery section of the Launcher screen and a destination (i.e. folder)
+for the location of the cloned repos.
+
+Several repositories can be configured to display as individual gallery tiles. Under the `exhibits`
+section, each repository must have a `title` and a `git` location. There are also several other
+optional config options.
```

Comment:
Just a note: currently the tiles are designed so that they are most beautiful when there is a description. I would personally say that adding description is recommended, but do not want to push it either.